apps/auth-server/routes/routes.js

```diff
@@ -155,6 +155,14 @@ module.exports = function (app) {
 
     app.get('/', authLocal.index);
 
+    app.get('/health', (req, res) => {
+      res.status(200).json({
+        status: 'UP',
+        message: 'Server is healthy',
+        timestamp: new Date().toISOString(),
+      });
+    });
```

Comment:
Shouldn't this be before the other app.use's? In the other apps, you also have this before the app.get('/') right?